go/engine/account_reset.go

```diff
@@ -215,12 +212,19 @@ func (e *AccountReset) resetPrompt(m libkb.MetaContext, eventType int, readyTime
 		return nil
 	}
 
-	// Notify the user how much time is left
-	if err := m.UIs().LoginUI.DisplayResetProgress(m.Ctx(), keybase1.DisplayResetProgressArg{
-		Text: fmt.Sprintf(
+	// Notify the user how much time is left / if they can already reset
```

Comment:
I don't understand, they haven't reset their account yet, it's just a notification that they _can_ perform if if they get into the flow using `keybase login` rather than `keybase account reset-start` (which we might remove).

Reply:
sorry, it's just that "can already" together just doesn't make sense.  either "they can reset" or "they already reset".

The notificationText "Your account reset is ready!" sounds a little weird to me too.  It's like they ordered an account reset at a restaurant and it's ready for pickup.  How about just "Log in to finish resetting your account." or something like that.

Reply:
@pzduniak `keybase account reset-start` will likely supersede `keybase account reset` command (and be exposed in prod)  but not be removed

Reply:
I've already fixed `keybase account reset-start` in this branch, wrt the wording I'll update it later, focusing on the actual broken tests now. I thought it sounded OK (I can find plenty of examples in newspapers etc.) but I'll trust your gut here.